Umweltzone/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,348 @@
+        Verder was OpenStreetMap sinds een aantal een belangrijke bron
+        zones met lage emissie zijn daar te vinden. Bovendien hebben sommige zones
+        handmatig gedigitaliseerd op basis van de officiële publicaties van de
+        Federaal Milieuagentschap.
+        Vanwege het feit dat veel autoriteiten afbeeldingen met een lage resolutie publiceren
+        van gescande kaarten kan niet worden gegarandeerd dat de zone precies grenst
+        overeenkomen met het pad in de realiteit. Zonegegevens en coördinaten verschillen per bron,
+        kwaliteit en precisie in het digitalisatieproces.
+        Bovendien is er geen automatisme om op de hoogte te worden gebracht van aangebrachte wijzigingen
+        door de steden. Zonegegevens worden handmatig bijgewerkt en zijn sterk afhankelijk van de
+        feedback van vriendelijke gebruikers zoals jij.
+        Om deze redenen kunnen we niet garanderen voor een perfecte visualisatie en
+        wij zijn niet aansprakelijk voor kosten of straffen. We raden ten zeerste aan om altijd
+        vertrouw op het verkeersbord in de betreffende steden.
+    </string>
+    <string name="appinfo_references_name_umweltbundesamt">Federaal Milieukantoor</string>
+
+    <!-- AboutActivity / Graphics -->
+    <string name="appinfo_references_graphics_title">Grafische vormgeving</string>
+    <string name="appinfo_references_graphics_content">
+        Afbeeldingen zoals het logo en stickers zijn afkomstig van Wikimedia Commons.
+    </string>
+
+    <!-- AboutActivity / Texts -->
+    <string name="appinfo_references_texts_title">Teksten</string>
+    <string name="appinfo_references_texts_content">
+        Een bronreferentie wordt gegeven wanneer tekst ergens anders vandaan komt.
+    </string>
+
+    <!-- AboutActivity / Libraries -->
+    <string name="appinfo_references_libraries_title">Bibliotheken</string>
+    <string name="appinfo_references_libraries_content">
+        De volgende software-bibliotheken worden binnen de toepassing gebruikt.
+    </string>
+
+    <!-- AboutActivity / Sourcecode -->
+    <string name="appinfo_sourcecode_title">Broncode</string>
+    <string name="appinfo_sourcecode_content">
+        De broncode van de applicatie is online beschikbaar.
+    </string>
+    <string name="appinfo_sourcecode_url_title">Applicatie broncode</string>
+
+    <!-- AboutActivity / License -->
+    <string name="appinfo_license_title">Licenties</string>
+    <string name="appinfo_license_content">
+        De broncode van de Umweltzone-applicatie valt onder de GPLv3-licentie.
+        De geometriecoördinaten uit OpenStreetMap zijn in licentie gegeven onder de
+        Open Database License (ODbL) v1.0. Andere geometriecoördinaten zijn gelicentieerd als
+        gedefinieerd door de genoemde gegevensbronnen. Afbeeldingen en teksten zijn gelicentieerd onder
+        de Creative Commons Attribution 3.0 Unported-licentie.
+    </string>
```

Comment:
Please check preview of the longer texts block in Android Studio in `activity_about.xml`. There is a couple of blanks in the texts. Once you scroll down in the preview you can see the text being rendered.